Automattic/jetpack
# Update Form child blocks to show Manage Responses section

Fixes #27091

#### Changes proposed in this Pull Request:
* Update Form child blocks to show the "Manage Responses" section on the Sidebar

![image](https://user-images.githubusercontent.com/7811225/198367115-9bd7b757-7836-44ee-b7cf-f1e7f14643b8.png)

#### Other information:


#### Jetpack product discussion
N/A

#### Does this pull request change what data or activity we track or use?
No

#### Testing instructions:
* Create a page/post and add a Form block
* Include all available child blocks to the form and inspect each of them to ensure the "Manage Responses" section is available on the Sidebar
